.github/workflows/CI.yml

```diff
@@ -92,15 +94,15 @@ jobs:
           toolchain: nightly
       - run: rustup component add rustfmt
       - run: cargo fmt --all -- --check
-    
+
   release:
     runs-on: ubuntu-latest
     name: release
     needs: [build, clippy, test, test-script, fmt]
-    if: github.ref == 'refs/heads/release'
+    if: github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main'
```

Comment:
why did `release` change to `main`, did the release strategy change?

Reply:
yeah, exactly. I was looking at your concerns in #71 and wanted to make things a bit more simpler!